### PR TITLE
feat: Preview data sample is added

### LINKED ISSUE #316: Show 5-6 unique values in data sample (including NA)
Instead of the first 5 rows, let's take the first 5-6 unique values in the sample we scan (500 rows).

### PR DESCRIPTION
**Pull request by philter87**
## Description

- This PR adds a feature. Shows up to 5 unique values for a column (see picture below for an example)

image

## Related Issues

Closes #316 

## Testing

- [X] Yes
- [ ] No, not needed (give a reason below)
- [ ] No, I need help writing them

## Reviewer Focus
This PR only needs a quick review.

## Checklist

For all PRs that are not general documentation

- [X] Tests accompany or reflect changes to the code
- [X] Tests ran and passed locally
- [X] Ran the linter and formatter
- [X] Build has passed locally
- [X] Relevant documentation has been updated

### CODE REVIEW

**Comment by signekb** [comment]
@philter87 From the screenshot you shared, I see that HEALTHCARE_EXPENSES shows five unique values and the text says `<5 unique values`. Should this be `<=5 unique values`?

File: sprout/templates/column-review.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h2 style="margin-bottom: 20px">
             </div>
 
             <div style="min-width: 0">
-              <p style="color: grey; font-size: 0.75rem; font-weight: bold; margin-bottom: 5px">Data preview:</p>
+              <p style="color: grey; font-size: 0.75rem; font-weight: bold; margin-bottom: 5px">Preview (<=5 unique values):</p>
```
Comment (signekb) [inline sprout/templates/column-review.html:91]:
```suggestion
              <p style="color: grey; font-size: 0.75rem; font-weight: bold; margin-bottom: 5px">Preview (showing up to 5 unique values):</p>
```

File: sprout/tests/tests_views_column_review.py [hunk 2]
```diff
@@ -32,7 +33,7 @@ def setUp(self):
             allow_duplicate_value=True,
         )
 
-        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C")
+        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C\n4,C")
```
Comment (signekb) [inline sprout/tests/tests_views_column_review.py:36]:
```suggestion
        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C\n4,C\n5,C\n6,C")
```
Might be nice to include that one of the test columns include more than 5 unique values.

File: sprout/tests/tests_views_column_review.py [hunk 3]
```diff
@@ -71,5 +72,14 @@ def test_column_review_view_post_valid_data(self):
         # Assert the status code
         self.assertEqual(response.status_code, 200)
 
+    def test_create_sample_of_unique_values(self):
+        """Test if correct sample values are created."""
+        table_metadata_id = self.table_metadata.id
+
+        sample = create_sample_of_unique_values(table_metadata_id)
+
+        self.assertEqual([1, 2, 3, 4], sample["TestColumn"])
```
Comment (signekb) [inline sprout/tests/tests_views_column_review.py:81]:
```suggestion
        self.assertEqual([1, 2, 3, 4, 5], sample["TestColumn"])
```

**Review by signekb — CHANGES_REQUESTED** [review]
Very nice! 🎉  I have added my preferred wording for the text above the data sample as well as added that one column in the test has more than 5 unique values.

**Review by signekb — DISMISSED** [review]
Very nice 🎉

**Review by lwjohnst86 — DISMISSED** [review]

**Review by lwjohnst86 — APPROVED** [review]
🎉

**Pull request merged**